### PR TITLE
[NHentai] Fix search with exact tag

### PR DESCRIPTION
**Pull request by ZIDOUZI**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/all/nhentai/build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ext {
     extName = 'NHentai'
     extClass = '.NHFactory'
-    extVersionCode = 42
+    extVersionCode = 44
```
Comment (vetleledaal) [inline src/all/nhentai/build.gradle]: IMO, should only bump this by 1, even if there's another active PR for this source.

Usually not an issue, but if for some reason the PRs can't be merged at the same time- well, then you should look into bumping it.

Comment (ZIDOUZI) [reply src/all/nhentai/build.gradle]: thank you for the pointer

**Review by AwkwardPeak7 — APPROVED** [review]

**Comment by mangkoran** [comment]
Hi I encountered issue with version 1.4.43. If I sort by popular today, this week, or this month (no other filter) only few results will be shown. This was not an issue in 1.4.42.

Below is sort by popular today.
Screenshot_2024-08-17-13-08-39-317_eu kanade tachiyomi yokai-edit

**Comment by ZIDOUZI** [comment]
> Hi I encountered issue with version 1.4.43. If I sort by popular today, this week, or this month (no other filter) only few results will be shown. This was not an issue in 1.4.42.

Will fixed soon

**Pull request merged**